### PR TITLE
updates features section

### LINKED ISSUE #1201: [Site] Enhance and update the features page
#### Current State of the Meshery.io site
The whole features page needs to be refactored in terms of UI and content. The content is outdated and needs to be updated with the latest features.

#### Desired State of the Meshery.io site
1. Refer to this feature page on layer5.io - https://layer5.io/cloud-native-management/meshery to implement the new for this site.
2. Also mention Meshery's extensibility features - Meshery Extension, Catalog, Snapshot. 

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by thisiskaransgit**
**Description**

- This PR fixes #1201 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by Chadha93 — CHANGES_REQUESTED** [review]
Thanks @thisiskaransgit, I recently few updates to the features on[ layer5](https://layer5.io/cloud-native-management/meshery) will you compare and update accordingly in this PR?

File: collections/_pages/features.html [hunk 1]
```diff
@@ -181,7 +178,51 @@ <h2 style="margin: 0">Use the <span class="special-text">Service Mesh Performanc
   </div>
   </div>
 </section>
+<section>
+  <div class="text">
+    <h2>The<span class="special-text">Extensible</span>Management Plane</h2>
+    <h3>Plug in different service meshes, load generators and providers.</h3>
+    <p>Meshery is not just an application. It is a set of microservices where the central component is itself called Meshery. Integrators may extend Meshery by taking advantage of designated Extension Points.
```
Comment (Savio629) [inline collections/_pages/features.html:184]: application, it is

**Comment by Savio629** [comment]
@thisiskaransgit Also I feel the cloud native catalog card images doesn't go well with the gray background
tempdel

**Comment by Ghat0tkach** [comment]
hii @thisiskaransgit 


Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Review by Chadha93 — COMMENTED** [review]
@thisiskaransgit Matched the features on Layer5.io?

**Comment by thisiskaransgit** [comment]
@Chadha93 I did

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@thisiskaransgit It's still different, please match the points from here - https://layer5.io/cloud-native-management/meshery
image

**Comment by thisiskaransgit** [comment]
@Chadha93, thanks for pointing out, two points were left to update, did that. Additionally, there was a design issue after updating the branch to the latest, which I fixed.

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**